src/App.js

```diff
@@ -9,16 +9,16 @@ import React from "react";
 const { setConfig, getValue } = Config;
 
 const testApiRoot =
-  "https://structuredcontentdev.bcg.ad.bcgov.us/api/hub/news/police/police";
+  "https://structuredcontentdev.bcg.ad.bcgov.us/api/hub/structuredContent/police";
 const prodApiRoot =
-  "https://services.baltimorecountymd.gov/hub/api/hub/news/police/police";
+  "https://services.baltimorecountymd.gov/hub/api/hub/structuredContent/police";
 
 // HACK - the Config utiltiy does not account for beta.
 // TODO: This will need to be addressed when we get closer to launch
 const localApiRoot =
   window.location.hostname.indexOf("beta") > -1
     ? testApiRoot
-    : "https://localhost:44393/api/hub/news/police/police";
+    : "https://localhost:44393/api/hub/structuredContent/police";
```

Comment:
`/structureContent/news/police` i believe. I think i may have told you the wrong url, thats why i wouldn't trust me. Would also recommend testing prior to submitting the pull request.

Reply:
done. test works